src/frontend/src/tests/instance_test.rs

```diff
@@ -1136,6 +1136,90 @@ async fn test_execute_copy_to_s3(instance: Arc<dyn MockInstance>) {
     }
 }
 
+#[apply(both_instances_cases)]
+async fn test_copy_table_to_fs_csv_issue_1519(instance: Arc<dyn MockInstance>) {
+    let instance = instance.frontend();
+    execute_sql(
+        &instance,
+        "create table demo(host string, cpu double, memory double, ts timestamp time index);",
+    )
+    .await;
+
+    let output = execute_sql(
+        &instance,
+        r#"insert into demo(host, cpu, memory, ts) values
+                    ('host1', 66.6, 1024, 1655276557000),
+                    ('host2', 88.8,  333.3, 1655276558000)
+                    "#,
+    )
+    .await;
+    assert!(matches!(output, Output::AffectedRows(2)));
+
+    let output = execute_sql(
+        &instance,
+        r#"Copy demo TO '/tmp/demo/export/csv/demo.csv' with (format='csv');
```

Comment:
Will this test create a directory under `/tmp/demo/export/`? If so, better to use tempdir to create a temp directory for this file.

Reply:
I removed these tests, and they were covered by sqlness.